### PR TITLE
Feat: scheduledAt in execution response model

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Adds scheduledAt date for scheduled executions (execute later). This is needed to then show this detail in Console.

## Test Plan

- [x] Update tests

## Related PRs and Issues

x

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/config/collections.php [hunk 1]
```diff
@@ -3848,6 +3848,39 @@
                 'array' => false,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('scheduledAt'),
```
Comment (Meldiron) [inline app/config/collections.php:3852]: Once Bradley's PR is merged, update response filter. BEFORE MERGING THIS!

File: src/Appwrite/Migration/Version/V16.php [hunk 2]
```diff
@@ -48,6 +48,36 @@ protected function migrateCollections(): void
             $this->projectDB->setNamespace("_{$this->project->getInternalId()}");
 
             switch ($id) {
+                case 'executions':
```
Comment (Meldiron) [inline src/Appwrite/Migration/Version/V16.php:51]: I dont think this is right file.

Comment (Meldiron) [reply src/Appwrite/Migration/Version/V16.php:51]: Indeed. Move it to V21.

File: tests/e2e/Services/Functions/FunctionsCustomClientTest.php [hunk 3]
```diff
@@ -269,6 +269,7 @@ public function testCreateScheduledExecution(): void
         // Schedule execution for the future
         \date_default_timezone_set('UTC');
         $futureTime = (new \DateTime())->add(new \DateInterval('PT10S'))->format('Y-m-d H:i:s');
+        $futureTimeIso = (new \DateTime($futureTime))->format('Y-m-d\TH:i:s.vP');
```
Comment (abnegate) [inline tests/e2e/Services/Functions/FunctionsCustomClientTest.php:272]: You can use `Utopia\Database\DateTime::formatTz` here

File: tests/e2e/Services/Functions/FunctionsCustomClientTest.php [hunk 4]
```diff
@@ -286,9 +287,24 @@ public function testCreateScheduledExecution(): void
...
 
         $executionId = $execution['body']['$id'];
 
+        // List executions and ensure it has schedule date
+        $response = $this->client->call(Client::METHOD_GET, '/functions/' . $function['body']['$id'] . '/executions', [
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey'],
+        ]);
+
```
Comment (abnegate) [inline tests/e2e/Services/Functions/FunctionsCustomClientTest.php:300]:
```suggestion
```

**Review by abnegate — APPROVED** [review]

**Pull request merged**